test/integration/api-support/test/index.test.js

```diff
@@ -118,16 +118,30 @@ function runTests(dev = false) {
   })
 
   it('should return error exceeded body limit', async () => {
-    const data = await fetchViaHTTP(appPort, '/api/parse', null, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json; charset=utf-8',
-      },
-      body: JSON.stringify(json),
-    })
+    let res
+    let error
+
+    try {
+      res = await fetchViaHTTP(appPort, '/api/parse', null, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json; charset=utf-8',
+        },
+        body: JSON.stringify(json),
+      })
+    } catch (err) {
+      error = err
+    }
 
-    expect(data.status).toEqual(413)
-    expect(data.statusText).toEqual('Body exceeded 1mb limit')
+    if (error) {
```

Comment:
Why is this broken on GH actions but not circle? This needs a TODO if it's an action item.

Reply:
It seems to be due to node's HTTP handling of closing the connection and happens inconsistently. It may be happening in GitHub actions more prevalently here vs CircleCi since we're running directly on ubuntu instead of in a  Docker container like in CircleCi

I updated this with a TODO to investigate further after the [mentioned issue](https://github.com/nodejs/node/issues/12339) is addressed in `node` or `node-fetch`